DataHubClient.js

```diff
@@ -60,12 +60,16 @@ export class DataHubClient {
    * @return {Promise<Object>} resolves to the inserted document.
    */
   async insert({doc}) {
+    if(!(doc.meta && typeof doc.meta === 'object')) {
```

Comment:
If `meta` is undefined, make it `{}`, otherwise, make sure it is an object (it cannot be an array):

`if(doc.meta === undefined) { doc.meta = {}; } else if(!(doc.meta && typeof doc.meta === 'object' && !Array.isArray(doc.meta)) { throw error }`

Reply:
Might want a helper function that validates meta for that `else if`.